learning_resources/tasks.py

```diff
@@ -430,3 +431,58 @@ def summarize_unprocessed_content(
         ]
     )
     return self.replace(summarizer_tasks)
+
+
+@app.task(bind=True)
+def scrape_marketing_pages(self):
+    """
+    Scrape marketing pages (for programs and courses)
+    and store them as content files if they dont exist
+    """
+    log.info("Running scrape_marketing_pages task")
+    resource_ids = set(
+        LearningResource.objects.filter(
+            published=True, resource_type__in=["course", "program"]
+        ).values_list("id", flat=True)
+    )
+
+    existing_page_resource_ids = set(
+        ContentFile.objects.filter(file_type="marketing_page").values_list(
+            "learning_resource_id", flat=True
+        )
+    )
+    missing_pages = list(resource_ids.difference(existing_page_resource_ids))
```

Comment:
One issue with this is that existing pages are never updated, so they could get out of date if the instructors change from one run to another for example.  Generally the ETL pipelines update everything even if something has changed or not.  That could be improved on at some point by comparing previous attribute values vs freshly retrieved values, but that seems like work for another PR.